### ISSUE TITLE
please add directly report feature in client

### ISSUE THREAD

**Issue by codev911** [idena-network/idena-desktop#275]
Dear idena team,

I want to suggest a feature that is very important and needs to be in the IDENA client. please add a report feature for bugs or the like experienced by the client. Because all this time reports have only been collected on the discord channel and in my opinion it is less effective. Maybe for my suggestion how to work for this feature is:

1. Obtain user IP address information
2. Getting what OS the user is using
3. Get the log file you want (example: node.log, error.log and output.log)
4. Let the user write complaints that are experienced in categories
5. accompanied by proof of image (image upload form)

it can all be sent directly from the client and you can find what problems occur more quickly and efficiently than before

that's all I want to say

thank you

**Comment by bingbinglee** [comment]
I would say, not "Report" since it has different meaning in the ecosystem but "Give Feedback" instead.

**Comment by codev911** [comment]
i agree with @codev911 , some people not familiar with github, and some people not hava good knowledge for operating computer, it will help other people to reach dev and tell their problem experience with idena client or node

**Comment by rioda-org** [comment]
"Report a problem" button

**Comment by RealMahmoud** [comment]
I think this is needed badly right now cause we are very near the 10k users ..

**Comment by rioda-org** [comment]
Something like this would help alot. Some button that would auto collect all logs and send it to some cloud store service, or even better on github if possible.
People fail validation, and are not realy motivated to "hack" to find all logs. I bet we need lot of "Late submission" logs to see if there is some greater problem there or just shitty routers or pc's, pardon my french...
